Weather/Views/DailyWeatherView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  DailyWeatherView.swift
+//  Weather
+//
+//  Created by Alex on 5.08.22.
+//
+
+import SwiftUI
+
+struct DailyWeatherView: View {
+    @ObservedObject var cityVM: CityViewModel
+    
+    var body: some View {
+        ForEach(cityVM.weather.daily) { weather in
+            LazyVStack {
+                dailyCell(weather: weather)
+            }
+        }
+    }
+    
+    private func dailyCell(weather: DailyWeatherData) -> some View {
+        HStack {
+            Text(cityVM.getDayFor(timeStamp: weather.dt).uppercased())
+                .frame(width: 50)
+            
+            Spacer()
+            
+            Text("\(cityVM.getTemperature(temp: weather.temp.max)) | \(cityVM.getTemperature(temp: weather.temp.min))°C")
+                .frame(width: 150)
+            
+            Spacer()
+            
+            cityVM.getWeatherIconFor(icon: weather.weather.count > 0 ? weather.weather[0].icon : "sun.max.fill")
+        }
+        .foregroundColor(.white)
+        .padding(.horizontal, 40)
+        .padding(.vertical, 15)
+        .background(RoundedRectangle(cornerRadius: 5).fill(LinearGradient(gradient: Gradient(colors: [Color.blue.opacity(0.5), Color.blue]), startPoint: .topLeading, endPoint: .bottomTrailing)))
```

Comment:
Please, break this line into several. It's very long

Reply:
Fixed.)